From 33c4929515bdc41ce3d6dda456bc6dd085d416eb Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Mon, 20 Sep 2021 20:35:08 +0200
Subject: [PATCH] Unbreak fontinst KCM

KCMs without JSON metadata must not be installed into <plugins>/kcms. They belong in <plugins> until they are ported

BUG: 442748
---
 kcms/kfontinst/kcmfontinst/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kcms/kfontinst/kcmfontinst/CMakeLists.txt b/kcms/kfontinst/kcmfontinst/CMakeLists.txt
index 8b5ae5d5e..e85045fd5 100644
--- a/kcms/kfontinst/kcmfontinst/CMakeLists.txt
+++ b/kcms/kfontinst/kcmfontinst/CMakeLists.txt
@@ -3,7 +3,7 @@ set(kcm_fontinst_PART_SRCS ${libkfontinstview_SRCS} ${libkfontinstjobrunner_SRCS
     FontFilter.cpp FontFilterProxyStyle.cpp FcQuery.cpp PreviewList.cpp)
 # qt_add_dbus_interface(kcm_fontinst_PART_SRCS ../dbus/org.kde.fontinst.xml FontinstIface)
 
-kcoreaddons_add_plugin(kcm_fontinst SOURCES ${kcm_fontinst_PART_SRCS} INSTALL_NAMESPACE "kcms")
+add_library(kcm_fontinst MODULE ${kcm_fontinst_PART_SRCS})
 
 target_link_libraries(kcm_fontinst
     KF5::Archive
-- 
GitLab

